packages/shell-api/package.json

```diff
@@ -13,7 +23,8 @@
   },
   "scripts": {
     "compile": "tsc -p tsconfig.json",
-    "pretest": "npm run compile",
+    "api-generate": "api-extractor run && ts-node bin/api-postprocess.ts",
+    "pretest": "npm run compile && npm run api-generate",
```

Comment:
Any reason not to make this part of `compile`? You'd also need this when building the mongosh executable, for example, not just for testing or publishing

Reply:
No good reason. I'll change it.